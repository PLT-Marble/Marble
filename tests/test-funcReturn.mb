int function get1(){
    return 1;
}

float function getPi(){
    return 3.14;
}

bool function getTrue(){
    return true;
}

matrix function getMatrix(){
    matrix a = [1.1, 2.2; 3.3, 4.4];
    return a;
}

main(){
    print(get1());
    printf(getPi());
    printb(getTrue());
    matrix a = getMatrix();
    printmf(a);
    return 0;
}