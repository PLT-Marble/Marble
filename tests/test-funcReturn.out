1
3.14
1
1.100000 2.200000